### DISCUSSION TITLE
Sound configuration

### DISCUSSION THREAD

**Discussion by szneqz** [alvr-org/ALVR#2239] [Q&A] [answered]
I have some strange problem with ALVR. I'm connected via usb cable to my laptop and mostly everything works fine but sometimes when I connect to my laptop sound is duplicated on headset and laptop. I don't know why and how to fix it (I can't find any good tutorial about sound configuration either).
My configuration on ALVR: 
input - VB cable
output - default device (my normal laptop speakers)

On steamvr configuration is default: input and output set to headset default and sound duplication disabled.

Thanks in advance for any responses :>

**Comment by SomeWaffle** [comment, marked as answer]
You could try using Voicemeeter, but I don't know if It'd work, you could try muting your laptop speakers? If that doesn't work, well then I am not fully sure :<

**Reply by szneqz** [reply]
Hi! Thanks for answer!
Anyway after that time I switched to linux and fixed it that way. I appreciate your commitment :>

**Reply by SomeWaffle** [reply]
No problem, I just saw your issue since I just got ALVR myself so I thought I could help, but I'm glad you've found a solution ^^
